nova/conductor/tasks/live_migrate.py

```diff
@@ -379,11 +380,8 @@ def _get_request_spec_for_select_destinations(self, attempted_hosts=None):
             request_spec.reset_forced_destinations()
             # NOTE(jkulik): We need the instance's current host in at least one
             # filter to make sure we don't pass vCenter boundaries, i.e. shards
```

Comment:
Should we update that note? It was written when `scheduler_hints` only contained `source_host` and `source_node`. Now that we specify `_nova_check_type`, too, this sounds a little out-of-date.